dask/array/tests/test_fft.py

```diff
@@ -108,3 +110,32 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+def test_wrap_bad_kind():
+    with pytest.raises(ValueError):
+        fft_wrapper(npfft.fft, kind="foo")
+
+
+@pytest.mark.parametrize("modname", ["numpy.fft", "scipy.fftpack"])
+@pytest.mark.parametrize(
+    "funcname",
+    ["fft", "ifft", "rfft", "irfft", "hfft", "ihfft"]
+)
+def test_wrap_ffts(modname, funcname):
+    fft_mod = pytest.importorskip(modname)
+    try:
+        func = getattr(fft_mod, funcname)
+    except AttributeError:
+        pytest.skip("`%s` missing function `%s`." % (modname, funcname))
+
+    if modname == "scipy.fftpack" and "rfft" in funcname:
+        pytest.skip(
+            "`%s` function `%s` doesn't support the NumPy API." % (
+                modname, funcname
+            )
+        )
```

Comment:
No because as noted in this [comment]( https://github.com/dask/dask/pull/2093#discussion_r106466514 ) `rfft` and  `irfft` do exist in SciPy, but they format data in a totally different way than NumPy's `rfft` and  `irfft`. So we have to massage it into a usable format if we want to test wrapping this function. Not sure it is worth it IMHO.